metron-interface/metron-alerts/README.md

```diff
@@ -6,15 +6,74 @@
 - [Installing on an existing Cluster](#installing-on-an-existing-cluster)
 
 ## Caveats
-* UI doesn't have an authentication module yet
 * UI uses local storage to save all the data.  A middleware needs to be designed and developed for persisting the data
 
 ## Prerequisites
-* Elastic search should be up and running and should have some alerts populated by metron topologies
+* The Metron REST application should be up and running and Elasticsearch should have some alerts populated by Metron topologies
+* The Management UI should be installed (which includes [Express](https://expressjs.com/))
 * The alerts can be populated using Quick Dev, Full Dev  or any other setup
 * UI is developed using angular4 and uses angular-cli
 * node.JS >= 7.8.0
 
+## Installation
+
+### From Source
+
+1. Package the application with Maven:
+
+    ```
+    cd metron-interface/metron-alerts
+    mvn clean package
+    ```
+
+1. Untar the archive in the $METRON_HOME directory.  The directory structure will look like:
+
+    ```
+    bin
+      metron-alerts-ui
+    web
+      expressjs
+        alerts-server.js
+      alerts-ui
+        web assets (html, css, js, ...)
+    ```
+
+1. Copy the `$METRON_HOME/bin/metron-alerts-ui` script to `/etc/init.d/metron-alerts-ui`
+
+1. [Express](https://expressjs.com/) is installed at `$METRON_HOME/web/expressjs/` as part of the Management UI installation process.  The Management UI should be installeed first on the same host as the Alerts UI.
+
+### From Package Manager
```

Comment:
RPM sounds like it exists in one of the followon PRs that's coming.  Can we move this section to that PR?